src/archiefvernietigingscomponent/destruction/views.py

```diff
@@ -348,6 +364,23 @@ def get_context_data(self, **kwargs):
                 },
             }
         )
+
+        # If the current reviewer has previously requested changes,
+        # add any comments that the user has made when addressing those changes
+        current_reviewer = self.request.user
+        reviews = DestructionListReview.objects.filter(
+            destruction_list=destruction_list, author=current_reviewer
+        )
+
+        if reviews.exists():
+            last_review = reviews.order_by("created").last()
```

Comment:
I think it is possible to reuse `DestructionList.last_review()` property here
`last review = destruction_list.last_review()`

Reply:
I think that the filter on `author` may be important in the following case:

1. Record manager assigns 2 reviewers.
2. In the first review process, reviewer 1 doesn't ask for changes, reviewer 2 asks for changes.
3. The record manager makes changes and responds with a comment to the remarks of reviewer 2 
4. The review process restarts, but this time reviewer 1 also requests changes.
5. When reviewer 2 eventually can review again, if we only take the latest review, they won't be able to see the response to their initial review.

... but maybe i am overthinking this :sweat_smile:  What do you think?

Reply:
Omg, I haven't thought of it. I think you are right. 

But I think this logic can be moved to models, for example, an optional argument `reviewer` can be added to `DestructionList.last_review()` method.  But it's up to you. I don't insist in this case, just the views are getting fatter and it bothers me.

Reply:
I added an issue for this: https://github.com/maykinmedia/archiefvernietigingscomponent/issues/160